apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -19,9 +26,54 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | >=14.5        | >=84               | >=16    |
 
 FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will not be providing our own polyfills for features that we expect our browser support matrix to already support.
+There are often well-tested third-party polyfills available that applications can use to fill in the gaps.
+Consumers who require the use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. ResizeObserver),
+will need to implement or use a third-party polyfill to provide the functionality. See the partial support section below for features
+that may need to be polyfilled.
```

Comment:
It seems strange to start talking about Polyfills here in the full support matrix, shouldn't we talk about it after introducing the partial support, where polyfills start to be expected (and of course if you need support beyond partial you would need to hopefully be able to polyfill further)

Reply:
I'd suggest having a proper section on polyfills down below the partial browser matrix

Reply:
Added a `Polyfills` section in the document